src/scss/components/_notices.scss

```diff
@@ -95,6 +95,7 @@ $snackbar-box-shadow: $notices-box-shadow !default;
         }
     }
     .notification {
+        pointer-events: auto;
```

Comment:
was the close button the default of target of pointer event?

Reply:
pointer events were not being triggered since .notification was inheriting pointer-events: none from .notice.